packages/svelte/src/compiler/types/template.d.ts

```diff
@@ -303,6 +303,7 @@ export namespace AST {
 			/** `true` if contains a SpreadAttribute */
 			has_spread: boolean;
 			scoped: boolean;
+			path?: SvelteNode[];
```

Comment:
It's not really optional, isn't it? It's always set. This could be a bit confusing on the consumer side (in fact the if condition is unnecessariy checking for path existing). The one drawback is that you'd have to initialize it during AST creation, but I think that's fine.

Reply:
Well it's not initialized until the analysis so that's why I went this route but I can initialize it with an empty array but maybe that could be more confusing?

Reply:
None of these methods are initialized with their true value until the analysis phase so I think it's fine
```suggestion
			path: SvelteNode[];
```